database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -49,6 +49,13 @@ trait WorkflowMetadataSummaryEntryComponent {
       if workflowMetadataSummaryEntry.workflowExecutionUuid === workflowExecutionUuid
     } yield workflowMetadataSummaryEntry)
 
+  val workflowMetadataSummaryEntryExistsForWorkflowExecutionUuid = Compiled(
```

Comment:
This looks like it's just `val workflowMetadataSummaryEntryExistsForWorkflowExecutionUuid = workflowMetadataSummaryEntriesForWorkflowExecutionUuid.exists`